tests/test_shell_formatter.py

```
@@ -0,0 +1,273 @@
+    unformatted = """
+        awk '{{print $1}}' {input}
+    """
+    expected = """\
+awk '{{print $1}}' {input}
+"""
+    assert format_shell_code(unformatted) == expected
+
+
+def test_format_shell_code_many_variables():
+    # Tests that __SNAKEFMT_VAR_1__ doesn't accidentally replace __SNAKEFMT_VAR_10__
+    vars = [f"{{var{i}}}" for i in range(15)]
```

Ruff (0.15.12)

[error] Variable `vars` is shadowing a Python builtin

(A001)